### PR TITLE
Ignore rules other than `string` rules or `Rule` objects

### PR DESCRIPTION
**Pull request by andronikigr**
This Pull Request will resolve the issue here: https://github.com/knuckleswtf/scribe/issues/212

@shalvah 
I follow the suggestions you made in your answer to my question.

### CODE REVIEW

File: src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php [hunk 1]
```diff
@@ -196,6 +196,7 @@ protected function normaliseRules(array $rules)
         $testData = [];
         foreach ($rules as $key => $ruleset) {
             if (!Str::contains($key, '.*')) continue;
+            if (!is_string($ruleset) && !($ruleset instanceof Rule)) continue;
```
Comment (shalvah) [inline src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php:199]: Did you test this? I think the `ruleset` here is actually the group of all rules, which should be an array. I believe this should go in `parseRule()`, the function that parses each rule individually. Even better, this method (where therre's already a check for `Rule` class):

https://github.com/knuckleswtf/scribe/blob/73351b170450d69e4039edd58034a883d4b0648e/src/Extracting/Strategies/BodyParameters/GetFromFormRequest.php#L399-L409

**Review by shalvah — CHANGES_REQUESTED** [review]

**Comment by WesleyGoncalves** [comment]
The PR changes worked for me.

**Comment by shalvah** [comment]
> It actually worked for the first time I executed the script. Second time, it stops with this error
> 
> > rmdir(/path/to/project_dir/public/vendor/scribe/images): Directory not empty

That's an unrelated error; should have been fixed in recent versions.

**Comment by shalvah** [comment]
> The PR changes worked for me.

Also, yes, the PR changes will work for you in the sense that Scribe won't crash, but _all_ your validation rules will be ignored. You can check and confirm that.

Closing this PR for the real fix.

**Pull request closed without merging**